ast/edit/option_editor.go

```diff
@@ -0,0 +1,153 @@
+package edit
+
+import (
+	"fmt"
+
+	"github.com/influxdata/flux/ast"
+	"github.com/influxdata/flux/semantic"
+	"github.com/influxdata/flux/values"
+)
+
+type EditFn func(opt *ast.OptionStatement) (ast.Expression, error)
```

Comment:
Can you add a comment to this type? Also it should have `Option` in its name and it does not need `edit` in its name since we are in the `edit` package. Maybe `OptionFn`?